tests/accuracy/test_harpy.py

```diff
@@ -124,14 +149,17 @@ def _get_model_dataframe():
 
 def _write_tbt_file(model, dir_path):
     ints = np.arange(NTURNS) - NTURNS / 2
-    data_x = model.loc[:, "AMPX"].values[:, None] * np.cos(
-        2 * np.pi * (model.loc[:, "MUX"].values[:, None] +
-                     model.loc[:, "TUNEX"].values[:, None] * ints[None, :]))
-    data_y = model.loc[:, "AMPY"].values[:, None] * np.cos(
-        2 * np.pi * (model.loc[:, "MUY"].values[:, None] +
-                     model.loc[:, "TUNEY"].values[:, None] * ints[None, :]))
+    data_x = model.loc[:, "AMPX"].to_numpy()[:, None] * np.cos(
+        2 * np.pi * (model.loc[:, "MUX"].to_numpy()[:, None] +
+                     model.loc[:, "TUNEX"].to_numpy()[:, None] * ints[None, :]))
+    data_y = model.loc[:, "AMPY"].to_numpy()[:, None] * np.cos(
+        2 * np.pi * (model.loc[:, "MUY"].to_numpy()[:, None] +
+                     model.loc[:, "TUNEY"].to_numpy()[:, None] * ints[None, :]))
```

Comment:
wouldn't that be something for {plane: formulas for plane in PLANES} ?

Reply:
that is not really part of this PR...just changed .values to .to_numpy() here. I added the data_z, which would on contrary break the possible convention.